python/src/sdf/pyMaterial.cc

```diff
@@ -0,0 +1,127 @@
+    .def("emissive", &sdf::Material::Emissive,
+         "Get the emissive color. The emissive color is "
+         "specified by a set of three numbers representing red/green/blue, "
+         "each in the range of [0,1].")
+    .def("set_emissive", &sdf::Material::SetEmissive,
+         "Set the emissive color. The emissive color is "
+         "specified by a set of three numbers representing red/green/blue, "
+         "each in the range of [0,1].")
+    .def("render_order", &sdf::Material::RenderOrder,
+         "Get render order for coplanar polygons. The higher value will "
+         "be rendered on top of the other coplanar polygons. The default value "
+         "is zero.")
+    .def("set_render_order", &sdf::Material::SetRenderOrder,
+         "Set render order.")
+    .def("lighting", &sdf::Material::Lighting,
+         "Get whether dynamic lighting is enabled. The default "
+         "value is true.")
+    .def("set_lighting", &sdf::Material::SetLighting,
+        "Set whether dynamic lighting is enabled.")
+    .def("double_sided", &sdf::Material::DoubleSided,
+         "Get whether double sided material is enabled. The default "
+         "value is false.")
+    .def("set_double_sided", &sdf::Material::SetDoubleSided,
+         "Set whether double sided material is enabled.")
+    .def("script_uri", &sdf::Material::ScriptUri,
+         "Get the URI of the material script, if one has been set.")
+    .def("set_script_uri", &sdf::Material::SetScriptUri,
+         "Set the URI of the material script.")
+    .def("script_name", &sdf::Material::ScriptName,
+         "Get the name of the material script, or empty if one has not "
+         "been specified. The name should match an "
+         "script element in the script located at the ScriptUri().")
+    .def("set_script_name", &sdf::Material::SetScriptName,
+         "Set the name of the material script. The name should match an "
+         "script element in the script located at the ScriptUri().")
+    .def("shader", &sdf::Material::Shader,
+         "Get the type of shader.")
+    .def("set_shader", &sdf::Material::SetShader,
+         "Set the type of shader.")
+    .def("normal_map", &sdf::Material::NormalMap,
+         "Get the normal map filename. This will be an empty string if "
+         "a normal map has not been set.")
+    .def("set_normal_map", &sdf::Material::SetNormalMap,
+         "Set the normal map filename.")
+    .def("file_path", &sdf::Material::FilePath,
+         "The path to the file where this element was loaded from.")
+    .def("set_file_path", &sdf::Material::SetFilePath,
+         "Set the path to the file where this element was loaded from.")
+    .def("__copy__", [](const sdf::Material &self) {
+      return sdf::Material(self);
+    })
+    .def("__deepcopy__", [](const sdf::Material &self, pybind11::dict) {
+      return sdf::Material(self);
+    }, "memo"_a);
+
+    pybind11::enum_<sdf::ShaderType>(materialModule, "ShaderType")
```

Comment:
This enum is in the `sdf` namespace in C++, but under `sdformat.Material` in python. Should we make them consistent?